carma_ros2_utils/CMakeLists.txt

```diff
@@ -5,50 +5,46 @@ find_package(carma_cmake_common REQUIRED)
 carma_check_ros_version(2)
```

Comment:
Do we still want to keep this, considering there are ros1 packages in carma_utils?

Reply:
Yes, this is a different package.
You'll notice carma_utils is still in the source tree whereas this is carma_ros2_utils.